aisdc/attacks/worst_case_attack.py

```diff
@@ -52,7 +54,17 @@ def __init__(self, **kwargs):
             "min_samples_leaf": 10,
             "max_depth": 5,
         }
+        self.__dict__["json_file"] = None
         self.__dict__.update(kwargs)
+        # Reading parameters from a json file
+        if self.__dict__["json_file"] is not None:
+            print("First If passed")
```

Comment:
If this print statement is needed it would be best put in a logger?

Reply:
Supposed to remove it as I put it to do some test and removed it now.